frontends/precinct-scanner/src/screens/poll_worker_screen.tsx

```diff
@@ -523,6 +527,23 @@ export function PollWorkerScreen({
       </React.Fragment>
     );
   }
+
+  // we don't need to ever set isReplacingBallotBag to false
+  // because we depend on unmounting for that to happen.
+  if (isReplacingBallotBag) {
+    return (
+      <ReplaceBallotBagScreen
+        scannedBallotCount={scannedBallotCount}
+        pollWorkerAuthenticated
+        onComplete={() => {
+          if (onReplaceBallotBag) {
+            onReplaceBallotBag();
+          }
```

Comment:
note to self: `onComplete` via the PW screen (not forced ballot bag change, voluntary one) simply updates the app state. it is poll worker removing card which will unmount this PW screen and reset this.